c2rust-transpile/tests/snapshots.rs

```diff
@@ -36,8 +36,8 @@ fn config() -> TranspilerConfig {
         enabled_warnings: Default::default(),
         emit_no_std: false,
         output_dir: None,
-        translate_const_macros: false,
-        translate_fn_macros: false,
+        translate_const_macros: TranslateMacros::Minimal,
+        translate_fn_macros: TranslateMacros::Minimal,
```

Comment:
This should still match the default (at this point, `None`). We should change the settings used for snapshots at the same time as we change the settings used for command-line transpilation.

Reply:
I had left `test_translator.py` testing `experimental` as that's its previous behavior.

Reply:
> I mean that the default for snapshots and the CLI should be kept in sync. It's up to you whether we change that default (from `None` to `Minimal`/`Conservative`) in this PR or later.

Well we need to change it for snapshots in this PR in order to test it.  So I guess that means immediately changing the behavior for the CLI, which I can do.